src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -764,15 +844,23 @@ def _add_optional_tags_to_openai_json(
         index: int,
         add_model_name: bool,
         add_stream: bool,
+        extra_inputs: Dict,
+        output_tokens_mean,
+        output_tokens_stddev,
         model_name: str = "",
-        extra_inputs: Dict = {},
     ) -> Dict:
+        row = pa_json["data"][index]["payload"][0]
         if add_model_name:
-            pa_json["data"][index]["payload"][0]["model"] = model_name
+            row["model"] = model_name
         if add_stream:
-            pa_json["data"][index]["payload"][0]["stream"] = True
+            row["stream"] = True
+        if output_tokens_mean != LlmInputs.DEFAULT_OUTPUT_TOKENS_MEAN:
```

Comment:
do you need LlmInputs. here?
its all the same file no?

Reply:
You do. You could prepend "self." in regular methods. These are class methods, so we would need to prepend "cls." if we wanted to use that instead. That could be a good thing to target in a refactor for readability. Right now, this PR aims to be consistent with the rest of the class, which prepends LLMInputs before function calls and class variables.